### PR TITLE
Feed template

### PR DESCRIPTION
**Pull request by Noy-Maimon**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: demisto/etc#29086

## Description
Adding the feed template to init, as in: `demisto-sdk init -integration -t FeedHelloWorld` will create a feed integration repository and files.

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: demisto_sdk/commands/init/initiator.py [hunk 1]
```diff
@@ -556,13 +565,15 @@ def get_remote_templates(self, files_list, dir):
...
+        # create test_data dir
+        if self.template in [self.HELLO_WORLD_INTEGRATION] + self.DEFAULT_TEMPLATES\
+                + [self.HELLO_WORLD_FEED_INTEGRATION]:
             os.mkdir(os.path.join(self.full_output_path, self.TEST_DATA_DIR))
 
         if self.template in self.DEFAULT_TEMPLATES:
             pack_name = self.DEFAULT_TEMPLATE_PACK_NAME
 
-        elif self.template in self.HELLO_WORLD_TEMPLATES:
+        elif self.template in self.HELLO_WORLD_TEMPLATES + [self.HELLO_WORLD_FEED_INTEGRATION]:
```
Comment (yaakovi) [inline demisto_sdk/commands/init/initiator.py:576]: why `HELLO_WORLD_FEED_INTEGRATION` is not included in the `HELLO_WORLD_TEMPLATES` list?

File: requirements.txt [hunk 2]
```diff
@@ -41,3 +41,9 @@ urllib3>=1.25.7
 vulture>=1.3
 wcmatch>=5.1
 yamlordereddictloader>=0.4.0
+
+setuptools~=47.1.0
+pytest~=5.2.1
+python-dateutil~=2.8.1
+beautifulsoup4~=4.8.1
+pytz~=2020.1
```
Comment (yaakovi) [inline requirements.txt:49]: why did you add those?

File: .gitignore [hunk 3]
```diff
@@ -100,3 +100,8 @@ coverage_html_report
 # Unit Tests
 /demisto_sdk/commands/update_release_notes/tests_data/Packs/release_notes/1_1_0.md
 node_modules
+/demisto_sdk/commands/init/templates/FeedHelloWorld/CommonServerPython.py
+/demisto_sdk/commands/init/templates/FeedHelloWorld/CommonServerUserPython.py
+/demisto_sdk/commands/init/templates/FeedHelloWorld/conftest.py
+/demisto_sdk/commands/init/templates/FeedHelloWorld/demistomock.py
+/demisto_sdk/commands/init/templates/FeedHelloWorld/FeedHelloWorld_unified.yml
```
Comment (yaakovi) [inline .gitignore:107]: why do you need those?

**Review by yaakovi — COMMENTED** [review]
looks ok, check why the build fails (seems related to your requirements changes)

File: demisto_sdk/commands/init/templates/FeedHelloWorld/FeedHelloWorld.yml [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+  required: false
+  type: 0
+display: HelloWorld Feed
+name: HelloWorld Feed
+script:
+  commands:
+  - arguments:
+    - default: false
+      defaultValue: '10'
+      description: The maximum number of results to return. The default value is 10.
```
Comment (reutshal) [inline demisto_sdk/commands/init/templates/FeedHelloWorld/FeedHelloWorld.yml:117]: @roysagi Correct me if Im wrong, but the default value is added to the description automatically doesn't it?

**Review by reutshal — APPROVED** [review]

File: demisto_sdk/commands/init/templates/FeedHelloWorld/FeedHelloWorld.yml (hunk 4 above)
Comment (Noy-Maimon) [reply demisto_sdk/commands/init/templates/FeedHelloWorld/FeedHelloWorld.yml:117]: You are right - removed

**Review by jochman — APPROVED** [review]

File: CHANGELOG.md [hunk 5]
```diff
@@ -4,6 +4,7 @@
 # 1.3.1
 * Fixed an issue where the **validate** command failed to validate the release notes of beta integrations.
 * Updated the **upload** command to support indicator fields.
+* Added FeedHelloWorld template option for *--template* flag in **demisto-sdk init** command.
```
Comment (roysagi) [inline CHANGELOG.md:7]: note you are adding the RN on an existing version.
